src/client.js

```diff
@@ -223,6 +271,11 @@ IrcClient.prototype.quit = function(message) {
 };
 
 
+IrcClient.prototype.ping = function(message) {
+    this.raw('PING', message);
```

Comment:
This could do with a default `message` for people that don't care what message the ping sends. `message || '*'`